### PR TITLE
eETH-weETH Connectors

### PR DESCRIPTION
**Pull request by SamarendraGouda**
eETH Connector
Screenshot 2024-04-15 at 12 43 20 PM

weETH Connector
Screenshot 2024-04-15 at 1 13 03 PM

### CODE REVIEW

File: contracts/mainnet/connectors/eeth/helpers.sol [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "./interfaces.sol";
+
+contract Helpers{
+    IEtherfiPool constant etherfiPool = IEtherfiPool(0x308861A430be4cce5502d0A12724771Fc6DaF216);
```
Comment (shriyatyagii) [inline contracts/mainnet/connectors/eeth/helpers.sol:7]:
```suggestion
    IEtherfiPool internal constant ETHERFI_POOL = IEtherfiPool(0x308861A430be4cce5502d0A12724771Fc6DaF216);
```

File: contracts/mainnet/connectors/weeth/helpers.sol [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "./interfaces.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+
+contract Helpers{
+    IWEETH constant weETHContract = IWEETH(0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee);
```
Comment (shriyatyagii) [inline contracts/mainnet/connectors/weeth/helpers.sol:8]:
```suggestion
    IWEETH internal constant WEETH_CONTRACT = IWEETH(0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee);
```

File: contracts/mainnet/connectors/weeth/helpers.sol [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    TokenInterface constant eETHContract = TokenInterface(0x35fA164735182de50811E8e2E824cFb9B6118ac2);
```
Comment (shriyatyagii) [inline contracts/mainnet/connectors/weeth/helpers.sol:9]:
```suggestion
    TokenInterface internal constant EETH_CONTRACT = TokenInterface(0x35fA164735182de50811E8e2E824cFb9B6118ac2);
```

**Comment by SamarendraGouda** [comment]
Screenshot 2024-04-15 at 2 29 49 PM

**Comment by shriyatyagii** [comment]
Deployed contracts ->

Weeth connector - [0xE9C471bAd195Fc2206269BA4B1F001F93036e404](https://etherscan.io/address/0xE9C471bAd195Fc2206269BA4B1F001F93036e404#code)

**Comment by SamarendraGouda** [comment]
Screenshot 2024-04-15 at 3 27 15 PM

**Pull request merged**